extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/TableCommands.scala

```diff
@@ -594,6 +594,12 @@ object TableCommands extends CommandSpecs[TableCommandSpec] {
     TableCommandSpec(cmd, Seq(tableIdentDesc.copy(isInput = true)))
   }
 
+  val SetTableProperties = {
```

Comment:
It looks like the scope of SetTableProperties is beyond delta? Can we revise the PR tile

Reply:
Issue title should also revise?

Reply:
yeah